benchmarks/dynamo/ci_expected_accuracy/inductor_torchbench_dynamic_inference.csv

```diff
@@ -33,7 +33,7 @@ mobilenet_v2,pass,0
 mobilenet_v3_large,pass,0
 moco,pass,11
 nvidia_deeprecommender,pass,0
-opacus_cifar10,pass,45
+opacus_cifar10,pass,58
```

Comment:
Whomp whomp

Reply:
vaguely fake news, running perf now

Reply:
<img width="1088" alt="image" src="https://github.com/pytorch/pytorch/assets/4755252/d9d71b8b-4f9c-429d-abd7-9a48f78b7e6a">
Mostly fake, this model is well below 1.0 anyway, so idc